example/src/CheckoutView.js

```diff
@@ -86,42 +86,50 @@ const CheckoutView = () => {
     }
   };
 
+  const { config, paymentMethods, onConfigChanged } = usePaymentMethods();
+
+  useEffect(() => { onConfigChanged(config) },
+    [paymentMethods == null],
+  );
```

Comment:
This `useEffect` doesn't seem right 🤔  

In the dependency array you usually pass object instances or primitives, and in case those object references get updated, then the effect is triggered. In this current scenario, we are doing a direct comparison check here.

What is your end goal here? What is the use-case of this use effect?

Reply:
Fixed here [9608f81](https://github.com/Adyen/adyen-react-native/pull/56/commits/9608f811e255b756aee1474882d00fc2893ca9f4)